docs/README.es.md

```diff
@@ -0,0 +1,380 @@
+return finalResponse;
+```
+
+## Play Audio File
+Si tu skill esta registrado como reproductor de audio, puede enviar alguna de estas directivas (Instrucciones para reproducir, encolar or detener un audio).
+
+```csharp
+// crea una respuesta de habla
+string audioUrl = "http://mydomain.com/myaudiofile.mp3";
+string audioToken = "a token to describe the audio file"; 
+
+var audioResponse = ResponseBuilder.AudioPlayerPlay(PlayBehavior.ReplaceAll, audioUrl, audioToken);
+
+return audioResponse
+```
+
+# Variables de sesión
+Las variables de sesión son variables que pueden ser guardadas dentro de una respuesta y serán enviadas de ida y de regreso mientras la sesión permanezca abierta.
+## Respuesta con variables de sesión
+
+```csharp
+string speech = "The time is twelve twenty three.";
+Session session = input.Session;
+
+if(session.Attributes == null)
+    session.Attributes = new Dictionary<string, object>();
+session.Attributes["real_time"] = DateTime.Now;
+
+return ResponseBuilder.Tell(speech, session);
+```
+
+## Recuperando variables de sesión de una solicitud
+
+```csharp
+Session session = input.Session;
+DateTime lastTime = session.Attributes["real_time"] as DateTime;
+
+return ResponseBuilder.Tell("The last day you asked was at " + lastTime.DayOfWeek.ToString());
+```
+
+# Respuestas sin funciones auxiliares
+Si no quieres usar las funciones auxiliares Tell/Ask, Puedes construir la respuesta manualmente usando los objetos ```Response``` y ```IOutputSpeech```. Si quisieras incluir ```StandardCard``` o ```LinkAccountCard``` dentro de la respuesta, puedes hacerlo dentro del cuerpo de la repuesta:
+
+```csharp
+// crea la respuesta de habla
+var speech = new SsmlOutputSpeech();
+speech.Ssml = "<speak>Today is <say-as interpret-as=\"date\">????0922</say-as>.</speak>";
+
+// crea el doble aviso (reprompt)
+var repromptMessage = new PlainTextOutputSpeech();
+repromptMessage.Text = "Would you like to know what tomorrow is?";
+
+// crea el objecto del doble aviso (reprompt)
+var repromptBody = new Reprompt();
+repromptBody.OutputSpeech = repromptMessage;
+
+// crea la respuesta
+var responseBody = new ResponseBody();
+responseBody.OutputSpeech = speech;
+responseBody.ShouldEndSession = false; // esto desencadena el doble aviso
+responseBody.Reprompt = repromptBody;
+responseBody.Card = new SimpleCard {Title = "Test", Content = "Testing Alexa"};
+
+var skillResponse = new SkillResponse();
+skillResponse.Response = responseBody;
+skillResponse.Version = "1.0";
+
+return skillResponse;
+```
+# Respuestas progresivas
+Tu skill puede enviar respuestas progresivas para mantener al usuario enganchado mientras este prepara la respuesta completa o respuesta final de la petición. A continuación un ejemplo de una respuesta progresiva:
+```csharp
+var progressiveResponse = new ProgressiveResponse(skillRequest);
+progressiveResponse.SendSpeech("Please wait while I gather your data.");
+```
+
+## Contribuidores
```

Comment:
I think this will be hard to be kept up-to-date automatically...this section is generated by the all-contributors bot which can only edit one file, so additions here would get out of sync.  Should we just keep the contributors on the main EN version?